### ISSUE TITLE
[Feature request] Ctrl-F (find) in edit mode

### ISSUE THREAD

**Issue by matt32106** [Laverna/laverna#733]
I would like to be able to Ctrl-F in edit mode to trigger a search inside the currently edited note.

I wrote Ctrl-F because it is -more or less- standard in windows but it should be adapted per OS.

**Comment by funilrys** [comment]
Interesting :+1:

**Comment by yangliu28** [comment]
Agree, Ctrl-F in edit mode can come very handy, especially in the desktop app.

I can do Ctrl-F when using laverna in the browser, however it only searches the formatted text, not the editing area. Still it can be useful when viewing the notes in the browser.
